### DISCUSSION TITLE
TechKey AC1200 dual band wifi usb model 6B06 working

### DISCUSSION THREAD

**Discussion by 100kwatt** [chris1111/Wireless-USB-Big-Sur-Adapter#303] [General]
Big Sur 11.7.10
MacBook-Air ~ % kextstat
174    0 0xffffff7f9cd28000 0x2a2000   0x2a2000   com.realtek.driver.RtWlanU (1830.32.b27) DB501FB3-487F-382A-A630-86C151C13BFB <75 46 6 5 3 1>

https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/releases/download/V17/Wireless.USB.Big.Sur.Adapter-V17.zip

**Comment by chris1111** [comment]
Thanks [Done](https://github.com/chris1111/Wireless-USB-Big-Sur-Adapter/commit/a69d13585ff631a4178f410c9fdf5715b29ba8b8)
